plugins/lookup/tss.py

```diff
@@ -45,6 +45,15 @@
             - section: tss_lookup
               key: password
         required: true
+    domain:
+        default: ""
+        description: The domain with which to request the OAuth2 Access Grant.
+        env:
+            - name: TSS_DOMAIN
+        ini:
+            - section: tss_lookup
+              key: domain
+        required: false
```

Comment:
```suggestion
        required: false
        version_added: 3.6.0
```

Reply:
Thank you, fixed in 9a8284b58f7d0b8baed43dce9bcaa3439a34de77